src/utils/messages.ts

```diff
@@ -2,122 +2,123 @@ import { PendingTransactionMessages, TransactionType } from '../chain-data';
 
 export const messages = {
   // Dashboard errors
-  FAILED_TO_LOAD_DASHBOARD_DATA: 'Failed to load latest dashboard data',
+  FAILED_TO_LOAD_DASHBOARD_DATA: 'Failed to load latest dashboard data. Please try again later.',
 
   // Transaction messages
-  TX_APPROVAL_REJECTED: 'API3 token approval transaction rejected',
-  TX_APPROVAL_ERROR: 'Failed to approve API3 token allowance. Please try again',
-  TX_DEPOSIT_REJECTED: 'API3 token deposit transaction rejected',
-  TX_DEPOSIT_ERROR: 'Failed to deposit API3 tokens. Please try again',
-  TX_GENERIC_REJECTED: 'Transaction rejected',
-  TX_GENERIC_ERROR: 'An error has occurred. Please try again',
+  TX_APPROVAL_REJECTED: 'API3 token approval transaction rejected.',
+  TX_APPROVAL_ERROR: 'Failed to approve API3 token allowance. Please try again.',
+  TX_DEPOSIT_REJECTED: 'API3 token deposit transaction rejected.',
+  TX_DEPOSIT_ERROR: 'Failed to deposit API3 tokens. Please try again.',
+  TX_GENERIC_REJECTED: 'Transaction rejected.',
+  TX_GENERIC_ERROR: 'An error has occurred. Please try again.',
 
   // Validation messages
-  VALIDATION_INPUT_ZERO: 'Please ensure you have entered a non-zero amount',
-  VALIDATION_INPUT_PARSE: 'Unable to parse input amount',
-  VALIDATION_DEPOSIT_TOO_HIGH: 'Deposit value cannot be higher than the available balance',
-  VALIDATION_INPUT_TOO_HIGH: 'Input amount cannot be higher than the available balance',
+  VALIDATION_INPUT_ZERO: 'Please enter a value greater than 0.',
+  VALIDATION_INPUT_PARSE: 'Unable to parse input amount.',
+  VALIDATION_DEPOSIT_TOO_HIGH: 'Value too large. Please enter a value equal to or less than your available balance.',
+  VALIDATION_INPUT_TOO_HIGH: 'Value too large. Please enter a value equal to or less than your available balance.',
 
   // Blockchain errors
-  FAILED_TO_LOAD_CHAIN_DATA: 'Unable to load blockchain data',
-  FAILED_TO_LOAD_TREASURY_AND_DELEGATION: 'Unable to load delegation and treasury data',
+  FAILED_TO_LOAD_CHAIN_DATA: 'Unable to load blockchain data. Please try again later.',
+  FAILED_TO_LOAD_TREASURY_AND_DELEGATION: 'Unable to load delegation and treasury data. Please try again later.',
 
   // Delegation errors
-  UNABLE_TO_LOAD_DELEGATE: 'Unable to load the delagate of address',
-  INVALID_DELEGATE_ADDRESS: 'Delegation target must be a valid non zero address',
-  DELEGATE_IS_YOURSELF: "You can't delegate to yourself",
-  REDELEGATION_IS_FORBIDDEN: (targetDelegate: string) => `Address ${targetDelegate} delegates to some other account`,
+  UNABLE_TO_LOAD_DELEGATE: 'Unable to load the delegate of address. Please try again later.',
```

Comment:
This took me a couple times to read to understand. Maybe "Failed to verify that the intended delegate address is not already delegated. Please try again later."?

Reply:
Could also just be my though and my bad english

Reply:
I usually find @andreogle as "the god of wording messages", but this time I failed to understand that as well :D 

I like Tamara's suggestion.

Reply:
Sorry, I'll try again lol

I'm a user and I want to delegate my voting power to another address. I enter an address, hit the "Delegate" button and submit the transaction. I then get an error toast saying: "Unable to load the delegate of address. Please try again later.". It doesn't necessarily make sense at all since all I was doing is trying to delegate. Why is it trying to "load the delegate" at all?

So I just mean that the error should be clearer explaining that the delegate is already delegated, rather than something about the "loading failed"

Reply:
I like your explanation and error and don't have any other change to suggest except adding the word "that" for clarity!